### ISSUE TITLE
Wrong hourly prices from Entsoe 

### ISSUE THREAD

**Issue by gudnyh** [UtilitechAS/amsreader-firmware#463]
#**Describe the bug**
Prices NO1  on display do not correspond with Day Ahead NO1 prices

**To Reproduce**
1. Go to display on iPad in app
2. Scroll down to prices from Entso-e
4. See prices. Not prices for selected bidding zone and not equal to prices on Fortum-app

**Expected behavior**
Prices correspond to Day Ahead prices and those shown in Fortum app. As they have been previously.
![8E7D249A-6521-4508-9ED3-F17E86767165](https://user-images.githubusercontent.com/126117189/220764572-470a0717-9035-42dd-a3aa-9f23876c565c.png)
![62DE8825-3629-40B5-A5DF-8A2965387462](https://user-images.githubusercontent.com/126117189/220764776-d62144a2-9cc0-4425-bf6f-7e44482ef23e.png)
![0E840143-A430-4934-974A-5462905D7390](https://user-images.githubusercontent.com/126117189/220764785-0fbc4765-996c-40de-ac11-f02c3e8f9668.jpeg)

**Screenshots**
Attachment.

**Hardware information:**
 - Country: [e.g. Norway] Norway
 - Meter: [e.g. Aidon]
 - Encryption enabled [yes/no]
 - AMS reader: [e.g. Pow-U, ESP32 etc]
 - M-bus adapter (if applicable):

**Relevant firmware information:**
 - Version: [e.g. 2.1.0]
 - MQTT: [yes/no]
 - MQTT payload type: [e.g. JSON]
 - HAN GPIO: [e.g. GPIO5]
 - HAN baud and parity: [e.g. 2400 8E1]
 - Temperature sensors [e.g. 3xDS18B20]
 - ENTSO-E API enabled: [yes/no]

**Additional context**

**Comment by tronde-ams** [comment]
When I choose prices in EUR x 1000 they seems to correspond with https://transparency.entsoe.eu/transmission-domain/r2/dayAheadPrices/show

but the prices in NOK seems to be too low.

**Comment by gudnyh** [comment]
Prices on yesterday's attachment is  for 22.02, Wednesday. 

Target comparison is better with NordPool, this one:
https://www.nordpoolgroup.com/en/Market-data1/Dayahead/Area-Prices/ALL1/Hourly/?view=table

Prices on NordPool is pr MWH. Select NOK and /1000 to compare.

They do not correspond with those displayed on amsreader.

I take prices at NordPool as correct.

**Comment by ArnieO** [comment]
I cannot see any problem. This is data from https://www.nordpoolgroup.com/en/Market-data1/Dayahead/Area-Prices/ALL1/Hourly/?view=table
![image](https://user-images.githubusercontent.com/10295178/220871859-6774fe62-a212-4aef-9300-959ea5384c11.png)

When I compare with data in first post here, it seems to be identical.

**Comment by gudnyh** [comment]
Ok. I see the problem now. It's about readability of the price staples view:

It is difficult for the user to match hourly prices with what date they belong to.

Since there is no date in the view, and combined with the rolling window, the user easily mixes up part of the presented set of prices that belongs to the next day, and read them as today's prices.

I did not imagine until you verified the prices that the view worked like that. 

Means somehow the view probably makes a "jump" around 13 CET to show part of next day prices. This is a bit nerdy behavior that is difficult to grasp for other than people working close to the business - as I in fact do :-D - and yet - I did not read the view as you wanted me to :-o

I suggest you put in an indication/split for date or day name - or maybe shade "next day staples" to improve readability of the price view.
